src/components/Accordion/Accordion.tsx

```diff
@@ -160,7 +160,3 @@ const Accordion = React.forwardRef<FlatList, AccordionProps>(
     )
   },
 )
-
-Accordion.displayName = 'Accordion'
-
-export default React.memo(Accordion)
```

Comment:
should we keep React.memo ? 
We can use like below:
const MemoizedAccordion = React.memo(Accordion);
export { MemoizedAccordion };

Reply:
oke i'll keep memo at there. But i dont think the `MemoizedAccordion` is good. So i'll keep the name `Accordion`.
Do u agree with me?

Reply:
ok, it just example, you can change that name

Reply:
done